recipes/genform/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+$GXX ./src/*.cpp -o genform
```

Comment:
The build log is quite scrambled, so it's hard to catch, but the error is at
```
/Users/distiller/project/miniconda/conda-bld/genform_1563451308638/work/conda_build.sh: line 6: ./src/main.cpp: Permission denied
```
Which is probably because `GXX` is unset, thus this tries to execute `./src/*.ccp` with `./src/main.cpp` as the first entry for that glob.

Try `${CXX}` instead.